helix-view/src/view.rs

```diff
@@ -109,7 +112,12 @@ impl fmt::Debug for View {
 }
 
 impl View {
-    pub fn new(doc: DocumentId, gutter_types: Vec<crate::editor::GutterType>) -> Self {
+    pub fn new(
+        doc: DocumentId,
+        gutter_types: Vec<crate::editor::GutterType>,
+        gutter_line_numbers_width_min: usize,
+        gutter_line_numbers_width_max: usize,
```

Comment:
I've added parameters for the min and max line width, but I think this will get messy quick as more configuration options are added. Instead, think it would make more sense to pass a `Config` similar to how `Editor` accepts a `Config`. Looking for feedback before making changes.